src/WebUI/Pods/PodsTable.tsx

```diff
@@ -84,6 +90,18 @@ export class PodsTable extends BaseComponent<IPodsTableProperties> {
             renderCell: PodsTable._renderPodStatusCell
         });
 
+        if (showWorkloads) {
+            columns.push({
```

Comment:
about column sizes, i am assuming you will take care as per figma?

Reply:
Done